modules/ROOT/pages/_partials/data-sync/server-mqtt-pubsub.adoc

```diff
@@ -10,12 +10,13 @@ The xref:realtime-updates-{context}[] section describes how to set up the defaul
 
 == Using the MQTT PubSub mechanism
 
+// TODO: check link
 The https://npm.im/@aerogear/graphql-mqtt-subscriptions[`@aerogear/graphql-mqtt-subscriptions`] module provides an `AsyncIterator` interface used for xref:realtime-updates-{context}[implementing subscription resolvers]
 It connects the {sync-service} server to an MQTT broker to support horizontally scalable subscriptions.
 
 
 
-To initialize an MQTT client and pass it into `@aerogeaar/graphql-mqtt-subscriptions`
+Initialize an MQTT client and pass that client to the `@aerogeaar/graphql-mqtt-subscriptions` module, for example:
```

Comment:
Given the heading, and this sentence - applying the procedure template (https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc) to this content makes sense to me - wdyt?

Reply:
yeah, it's a procedure, but only one task, so I didn't number the task, does that make sense?

Reply:
It does make sense.